### PR TITLE
Register files.allancoding.is-a.dev & app.allancoding.is-a.dev

### PR DESCRIPTION
**Pull request by allancoding**
Added allancoding's subdomains:
- files.allancoding.is-a.dev
- app.allancoding.is-a.dev

### CODE REVIEW

**Comment by allancoding** [comment]
It has been 5 days. when will this be merged?
Thanks

**Comment by allancoding** [comment]
I just saw that merging has started again. Thanks for all your work, guys.

File: domains/files.allancoding.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "allancoding",
+        "email": "allancoding.dev@gmail.com"
+    },
+    "record": {
+        "CNAME": "allancoding.github.io"
```
Comment (DEV-DIBSTER) [inline domains/files.allancoding.json:7]: 404 CNAME record.

**Review by DEV-DIBSTER — CHANGES_REQUESTED** [review]

File: domains/files.allancoding.json (hunk 1 above)
Comment (allancoding) [reply domains/files.allancoding.json:7]: It is here: https://allancoding.github.io/files.allancoding.is-a.dev/

**Comment by allancoding** [comment]
I don't know why there is a 404 but this is the the github pages one: https://allancoding.github.io/files.allancoding.is-a.dev/

**Pull request closed without merging**